lib/chef/provider/log.rb

```diff
@@ -45,7 +45,7 @@ def load_current_resource
         # true:: Always return true
         def action_write
           Chef::Log.send(@new_resource.level, @new_resource.message)
-          @new_resource.updated_by_last_action(true)
+          @new_resource.updated_by_last_action(true) unless Chef::Config[:count_log_resource_updates] == false
```

Comment:
I hadn't put `default :count_log_resource_updates, true` earlier. That's why made the condition this way.
Anyway, will make the change